### PR TITLE
Bug 1876474 - Remove qemu guest requirement

### PR DESCRIPTION
**Pull request by lmandavi**
This PR is associated with the bug: https://bugzilla.redhat.com/show_bug.cgi?id=1876474
(QEMU guest agent does not pass the IP address)

Removed content stating that the QEMU guest agent is required to view the IP address for network interface cards.

CP to OCP 4.5, 4.6

### CODE REVIEW

**Comment by yaacov** [comment]
lgtm

**Comment by lmandavi** [comment]
HTML content:
https://bug-1876474-view-ip-address--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.html

**Review by bergerhoffer — COMMENTED** [review]
LGTM!

**Review by phoracek — CHANGES_REQUESTED** [review]

File: virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc
Comment (lmandavi) [reply virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc:7]: @yaacov - can you please advise/update?

Comment (phoracek) [reply virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc:7]: Yeah, that sounds good! If I'm ok with what is shown, I don't care, if I miss something in the report, I know where to look. :+1:

Comment (lmandavi) [reply virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc:7]: @yaacov @phoracek - please see the changed content.

1) In the topic on Installing the QEMU agent, I've adjusted the introductory sentence to indicate what the QEMU guest agent does.

2) In the topic on Viewing the IP address, I've adjusted the introductory paragraph stating that additional information can be viewed when the QEMU guest agent is running.

3) In the topic on Viewing the QEMU guest agent, I've adjusted the introductory paragraph to state that user can use the web console to view the info that the guest agent provides.

Comment (lmandavi) [reply virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc:7]: Topic on Installing the QEMU guest agent:
https://bug-1876474-view-ip-address--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/vm_networking/virt-installing-qemu-guest-agent.html

Topic on Viewing an IP address:
https://bug-1876474-view-ip-address--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/vm_networking/virt-viewing-qemu-guest-agent-web.html

Topic on Viewing the QEMU guest agent information:
https://bug-1876474-view-ip-address--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.html

**Comment by lmandavi** [comment]
@yaacov @phoracek - request you to approve this PR with lgtm for the changes, if no other changes are needed.

File: virt/virtual_machines/vm_networking/virt-installing-qemu-guest-agent.adoc
Comment (phoracek) [inline virt/virtual_machines/vm_networking/virt-installing-qemu-guest-agent.adoc:7]: It is still needed for secondary networks

Comment (phoracek) [inline virt/virtual_machines/vm_networking/virt-installing-qemu-guest-agent.adoc:7]: I just noticed this is under "Virtual machine networking" section. Could we move it out? It is not network specific.

File: virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc
Comment (phoracek) [inline virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc:7]: And information about secondary networks

File: virt/virtual_machines/vm_networking/virt-viewing-qemu-guest-agent-web.adoc
Comment (phoracek) [inline virt/virtual_machines/vm_networking/virt-viewing-qemu-guest-agent-web.adoc:7]: and secondary networks

**Review by phoracek — CHANGES_REQUESTED** [review]

File: virt/virtual_machines/vm_networking/virt-installing-qemu-guest-agent.adoc
Comment (lmandavi) [reply virt/virtual_machines/vm_networking/virt-installing-qemu-guest-agent.adoc:7]: Updated the content by adding secondary networks.

File: virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc
Comment (lmandavi) [reply virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc:7]: @phoracek - Please note. I am removing the second sentence because this entire module is focused only on viewing the IP address of the NIC and this content resides in the VM Networking section. A mention about viewing logged in users, file systems etc does not belong to this content module - all of that information will now be available in the Virtual Machines section.

All information about the QEMU guest agent (installing and viewing) is being moved to the Virtual Machines section.

File: virt/virtual_machines/vm_networking/virt-viewing-qemu-guest-agent-web.adoc
Comment (lmandavi) [reply virt/virtual_machines/vm_networking/virt-viewing-qemu-guest-agent-web.adoc:7]: Updated the content by adding info on secondary networks.

File: virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc
Comment (phoracek) [reply virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc:7]: Understood. Please let me know once it is moved and ready for another review

**Comment by lmandavi** [comment]
@phoracek - The QEMU guest agent now has a new home! 
https://bug-1876474-view-ip-address--ocpdocs.netlify.app/openshift-enterprise/latest/welcome/index.html

Note that two major topics have now been moved away from the networking section, and both of the QEMU guest agent topics are now available under virtual machines.

Please review and advise if this PR is now in LGTM status or if you spot any other changes! Thank you!

File: virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc
Comment (phoracek) [inline virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc:7]: Examples of IP reports we show in https://bug-1876474-view-ip-address--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.html won't be available without guest agent available. I know we cannot cross reference, but IMHO it would be worth mentioning guest agent at least by name so people connect it with the guest agent section.

**Comment by lmandavi** [comment]
@phoracek I've added a sentence in the module for IP address. Will add a link here when the html output is ready.

**Comment by lmandavi** [comment]
@phoracek - here's the updated content:
https://bug-1876474-view-ip-address--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/vm_networking/virt-
viewing-ip-of-vm-nic.html

File: virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc
Comment (phoracek) [inline virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc:7]: There is comma instead of full stop :eyes: 

Also, I don't think that  "users, file systems" is relevant here. Can we just say it shows additional info about VM's secondary networks?

Comment (lmandavi) [reply virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc]: Yes, content was updated to remove users, file systems.

Comment (lmandavi) [reply virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc]: @phoracek - please review.

**Review by phoracek — APPROVED** [review]
/lgtm

Looks good. Thanks!

**Review by bergerhoffer — COMMENTED** [review]
These changes LGTM. 

Just a question on making sure that there are redirects in place before this PR is merged.

File: _topic_map.yml
Comment (lmandavi) [reply _topic_map.yml:2342]: Redirect requested was submitted and acked by @vikram-redhat.

File: virt/virtual_machines/vm_networking/virt-installing-qemu-guest-agent.adoc
Comment (lmandavi) [reply virt/virtual_machines/vm_networking/virt-installing-qemu-guest-agent.adoc]: Content was moved out to new location.

File: virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc
Comment (lmandavi) [reply virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc]: Content was moved out to new location.

File: virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc
Comment (lmandavi) [reply virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.adoc]: Content was updated.

**Comment by vsibirsk** [comment]
lgtm

**Comment by lmandavi** [comment]
https://bug-1876474-view-ip-address--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/vm_networking/virt-viewing-ip-of-vm-nic.html

**Pull request merged**
